tests/manager/openAccount/managerCanChooseCurrencies.spec.js

```diff
@@ -12,4 +13,16 @@ Test:
 6. Select currency Rupee
 7. Assert the drop-dwon has value Rupee
 */
-});
\ No newline at end of file
+
+const openAccountPage = new OpenAccountPage(page);
+
+await openAccountPage.open();
+await openAccountPage.selectCurrencyFromDropdown('Dollar');
+await openAccountPage.assertDropdownHasValue('Dollar');
+
+await openAccountPage.selectCurrencyFromDropdown('Pound');
+await openAccountPage.assertDropdownHasValue('Pound');
+
+await openAccountPage.selectCurrencyFromDropdown('Rupee');
+await openAccountPage.assertDropdownHasValue('Rupee');
```

Comment:
The method `assertDropdownHasValue` may not work as intended for verifying the selected currency in a dropdown. Consider using `evaluate` to get the selected value or verify the selected option directly using `selectOption`.